examples/models/llama/export_llama_lib.py

```diff
@@ -1196,12 +1259,25 @@ def _get_source_transforms(  # noqa
         transformations based on the given checkpoint first. In those cases,
         this wil be a no-op.
         """
-        modelname = f"{modelname}_e"
+        # Create a mock args object with the necessary attributes
```

Comment:
Is it to limit the extend of changes? It's better than Namespace, but I'm wondering if eventually we need to group args, and there can still be passing the entire args even not all fields are needed.

Reply:
Yup this is just to limit the changes, after I do a final refactor of the quant source transform functions we can remove this